### ISSUE TITLE
Disabling click framework for Modals (Fancybox)

### ISSUE THREAD

**Issue by mcnejef** [Simbul/baker#466]
I've been trying to figure this out for a few hours now...is there a way to disable / enable portions (or all) of the click framework from javascript or?  I'm new to objective-C but a html/css/js veteran

Basic use case is in the use of a modal jQuery plugin like Fancybox for interactive content created in Hype or Adobe Edge or by hand.  I can get the modal to work, but the swipe and click handlers cause me pain (modal location stays pinned to particular location, swipe's and double clicks affect the primary page).  Fancybox supports callouts for the show / hide events where, in theory, I could disable some of the click-handling pieces of Baker on show and reinstate on hide.

Any thoughts?  Are there other methods I'm not thinking of to display non-book interactive content where I'd have total control over click handling?

Thanks for the help...

**Comment by ClovisMarsupial** [comment]
Hi, I don't know if that is exactly what you looking for but you could take a look at touchswipe. I'm using it and it fulfills my needs... http://labs.skinkers.com/touchSwipe/

**Comment by mcnejef** [comment]
Clovis, not quite what I was looking for.  I think it has more to do with the app container (the xcode project) constantly listening for swipe and double click events.  I have figured out a workaround (loosely, xcode is not my thang...).  

Inside of the shouldStartLoadWithRequest method in RootViewController.m I've modified the "PageNotFound" section to look like:

```
                if (page == NSNotFound)
                {
                    // ****** Internal link, but not one of the book pages --> load page anyway

                    NSLog(@"    Page not in index --> disabling swipes and double-clicks");
                    scrollView.scrollEnabled = NO;
                    doubleTapEnabled= NO;                        

                    return YES;
                }
                else
                {
                    scrollView.scrollEnabled = [[properties get:@"-baker-page-turn-swipe", nil] boolValue];
                    doubleTapEnabled= YES;
                }
```

And that "doubleTapEnabled" is a property I added along with a new condition inside of "userDidTap" that disables the double-click registration.  This effectively turns off click handling when you navigate to a page that isn't in the book manifest.  I do have one problem that swipes on these pages are still "stored" somehow, and the swipe is registered when navigation is returned to a book page.  Like if I link back to page 3, but I had two L->R swipes while on the NSNotFound page I'll see transitions that take me back to page 1.  I can describe this better if anyone wants to hear about it ;-)

So, I realize a better way of doing this would be to create my own custom link prefix and handler like, "noclick://page.html", but the method above works for me as I won't be including links to web-hosted pages.  I would however, still appreciate it if I could pop a modal from within a book page that 1)froze vertical scrolling, 2)stopped page swiping, 3) stopped double-clicks and 4) still allowed events to be captured by javascript (which I don't think would be a problem based my other experiences with the platform).

Thanks!
